From: Richard Acayan <mailingradian@gmail.com>
Date: Tue, 11 Mar 2025 22:11:00 -0400
Subject: [PATCH] arm64: dts: qcom: sdm670-google: enable cdsp

The Pixel 3a ships with DSP code that can perform math operations based
on a graph. Enable the CDSP to open up more computing power on this
device.

Signed-off-by: Richard Acayan <mailingradian@gmail.com>
---
 arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi b/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi
index 2b4c4f8..35089d1 100644
--- a/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi
@@ -833,6 +833,11 @@ cam_front_endpoint: endpoint {
 	};
 };
 
+&cdsp_pas {
+	status = "okay";
+	firmware-name = "qcom/sdm670/sargo/cdsp.mbn";
+};
+
 &gcc {
 	protected-clocks = <GCC_QSPI_CORE_CLK>,
 			   <GCC_QSPI_CORE_CLK_SRC>,
